### DISCUSSION TITLE
When/how does a color scheme get updated?

### DISCUSSION THREAD

**Discussion by fredrikaverpil** [wezterm/wezterm#4448] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20230712-072601-f4abf8fd

### Ask your question!

I'm using the excellent [tokyonight color scheme for wezterm](https://wezfurlong.org/wezterm/colorschemes/t/index.html#tokyo-night) and I noticed it [got updated the other day](https://github.com/folke/tokyonight.nvim/releases/tag/v2.8.0). But when or how does wezterm get ahold of this new version of the color scheme?

Do I have to wait until a new release of wezterm?

This is my relevant wezterm config:

```lua
local function get_appearance()
  if wezterm.gui then
    return wezterm.gui.get_appearance()
  end
  return "Dark"
end

local function scheme_for_appearance(appearance)
  if appearance:find("Dark") then
    return "Tokyo Night Moon"
  else
    return "Tokyo Night Day"
  end
end

config.color_scheme = scheme_for_appearance(get_appearance())
```

**Comment by bew** [comment, marked as answer]
You probably need to copy/link the updated colorscheme from the plugins' folder to a location wezterm can read, see https://wezfurlong.org/wezterm/config/appearance.html#defining-a-color-scheme-in-a-separate-file

**Reply by fredrikaverpil** [reply]
Allright, I could try that.

But I don't mind waiting, so I was just curious whether wezterm will fetch the latest of the bundled themes before shipping a new release too...

**Reply by nekowinston** [reply]
Yeah, it does get regularly updated for new releases: https://github.com/catppuccin/wezterm/issues/6#issuecomment-1186197439
